acquire/collector.py

```diff
@@ -331,13 +350,39 @@ def collect_glob(self, pattern: str, module_name: Optional[str] = None) -> None:
             else:
                 self.report.add_glob_collected(module_name, pattern)
 
+    def collect_symlink(
+        self,
+        path: Path,
+        module_name: Optional[str] = None,
+        follow: Optional[bool] = True,
+        volatile: Optional[bool] = False,
+    ) -> None:
+        try:
+            outpath = self.create_outpath(str(path))
+
+            self.output.write_bytes(outpath, b"", entry=path, size=0)
```

Comment:
The typehint for `entry` is `FilesystemEntry` but you pass in the `TargetPath` here.